contracts/services/ServiceTypeManager.sol

```diff
@@ -12,23 +12,6 @@ import "./IService.sol";
  * @author RIF protocols team
  */
 contract ServiceTypeManager is Ownable, ERC165Storage {
-    mapping(bytes4 => bool) private _supportedInterfaces;
```

Comment:
In case you are wondering why did I delete all of these lines of code, the reason is that the private `_supportedInterfaces` variable and the `supportsInterface` function are already implemented in the `ERC165Storage` abstract contract, there is no need to re-implement them again AFAIK